pkg/stub/handler.go

```diff
@@ -354,7 +395,15 @@ func (h *Handler) CreateDefaultResourceIfNeeded(cfg *v1.Config) (*v1.Config, err
 		cfg.Kind = "Config"
 		cfg.APIVersion = v1.GroupName + "/" + v1.Version
 		cfg = h.updateCfgArch(cfg)
-		cfg.Spec.ManagementState = operatorsv1api.Managed
+		switch {
+		// order of calls matters here ... bootstrap Removed when IPv6 regardless of disconnected checks
+		case util.IPv6Only():
+			cfg.Spec.ManagementState = operatorsv1api.Removed
+		case h.tbrInaccessible():
+			cfg.Spec.ManagementState = operatorsv1api.Removed
```

Comment:
So here's an idea - what if we just test for TBR access? The only reason we are bootstrapping `Removed` is that _today_ `registry.redhat.io` does not support IPv6 connections today. This improves the install experience by getting us around a whole host of failure modes for the samples operator - disconnected, crappy proxy, poor network, etc. Admins can always switch the samples operator to be `Managed` day 2.

Bonus - when `registry.redhat.io` does support IPv6, we don't need to make any code changes.

Reply:
I like the idea.  I'll sort through the implications in the other comments below.

Reply:
One fly in the ointment did just occur to me ... do we support proxy in IPv6 ?

Assuming so, I think we need to tweak the tbr check to include ipv6.

Reply:
@gabemontero I would assume that we do support proxy in IPv6